traffic_ops/traffic_ops_golang/config/config.go

```diff
@@ -173,6 +174,14 @@ type ConfigAcmeAccount struct {
 	HmacEncoded  string `json:"hmac_encoded"`
 }
 
+type DefaultCertificateInfo struct {
```

Comment:
Can we add some config validation to ensure that if the config contains the `default_certificate_info` section, that all the fields within it are non-empty? I'm not sure if that would cause the certificate generation to fail or not, but I'd think if there were any empty fields that it would be unintentional.